Resources/config/services.xml

```diff
@@ -210,10 +209,17 @@
             <tag name="sulu.context" context="admin"/>
         </service>
         <!--PROVIDER-->
-        <service id="sulu.core.localization_manager.core_provider" class="%sulu.core.localization_manager.core_provider.class%">
+        <!--<service id="sulu.core.localization_manager.core_provider" class="%sulu.core.localization_manager.core_provider.class%">-->
+            <!--<argument>%sulu_product.locales%</argument>-->
+
+            <!--<tag name="sulu.localization_provider" alias="HUHU"/>-->
+        <!--</service>-->
+        <service id="sulu_product.localization_provider" class="Sulu\Component\Localization\Provider\LocalizationProvider">
             <argument>%sulu_product.locales%</argument>
+
```

Comment:
this emtpy line actually is for the sake of better readability.
it's to clearly separate tags from arguments.
(i now did that for all tags in services.xml, which comes with the next commit)